openage/convert/value_object/read/media/smx.pyx

```diff
@@ -182,16 +593,20 @@ class SMX:
 
                 if layer_type is SMXLayerType.MAIN:
                     if layer_header.compression_type == 0x08:
-                        self.main_frames.append(SMXMainLayer8to5(layer_header, data))
+                        self.main_frames.append(
+                            SMXMainLayer8to5(layer_header, data))
 
                     elif layer_header.compression_type == 0x00:
-                        self.main_frames.append(SMXMainLayer4plus1(layer_header, data))
+                        self.main_frames.append(
+                            SMXMainLayer4plus1(layer_header, data))
 
                 elif layer_type is SMXLayerType.SHADOW:
-                    self.shadow_frames.append(SMXShadowLayer(layer_header, data))
+                    self.shadow_frames.append(
+                        SMXShadowLayer(layer_header, data))
 
                 elif layer_type is SMXLayerType.OUTLINE:
-                    self.outline_frames.append(SMXOutlineLayer(layer_header, data))
+                    self.outline_frames.append(
+                        SMXOutlineLayer(layer_header, data))
```

Comment:
These changes are not necessary. The line length was previously wel below 100 lines. Make sure yoe set the max line length in your autopep8 config to 100 too.